index.html

```diff
@@ -2030,6 +2030,111 @@ <h4>Presentations Using Derived Credentials</h4>
             </figcaption>
           </figure>
         </section>
+        <section>
+          <h4>Presentations Including Holder Claims</h4>
+          <p>
+A <a>holder</a> MAY use the <code>verifiableCredential</code> <a>property</a> in
+a <a>verifiable presentation</a> to include <a>verifiable credentials</a> from
+any <a>issuer</a>, including themselves. When the <a>issuer</a> of a
+<a>verifiable credential</a> is the <a>holder</a>, the <a>claims</a> in that
+<a>verifiable credential</a> are considered to be <em>self-asserted</em>.
+Such self-asserted claims can be secured by the same mechanism that secures
+the <a>verifiable presentation</a> in which they are included or by any
+mechanism usable for other <a>verifiable credentials</a>.
+          </p>
+          <p>
+The <a>subject(s)</a> of these self-asserted <a>claims</a> are not limited, so
+these <a>claims</a> can include statements about the <a>holder</a>, one of the
+other included <a>verifiable credentials</a>, or even the <a>verifiable
+presentation</a> in which the self-asserted <a>verifiable credential</a> is
+included. In each of these cases, the <code>CredentialSubject</code> would
+have an <code>id</code> <a>property</a> that matches the <code>id</code> of
+the <a>subject</a>, just as it is done in <a>verifiable credentials</a> that
+are not self-asserted.
+          </p>
+          <p>
+A <a>verifiable presentation</a> that includes a self-asserted
+<a>verifiable credential</a> that is only secured using the same mechanism as
+the <a>verifiable presentation</a> MUST include a <code>holder</code>
+<a>property</a>.
+          </p>
+          <p>
+All of the normative requirements defined for <a>verifiable credentials</a>
+apply to self-asserted <a>verifiable credentials</a>. Some additional
+considerations follow:
+          </p>
+
+        <dl>
+          <dt><var>type</var></dt>
+          <dd>
+It is RECOMMENDED that <code>SelfAssertedCredential</code> be included as an
```

Comment:
I agree with @selfissued and @dlongley that this is not necessary. You can just check to see if the key used to sign a VC belongs to the holder. For Data Integrity, you check to see if the `proof.verificationMethod` is associated with `holder` using the algorithm defined in https://w3c.github.io/vc-data-integrity/#retrieve-verification-method. I expect VC COSE JOSE would define the same sort of mapping. IOW, just because someone asserts this, doesn't mean it's true, you need to verify it... and if only a subset of the ecosystem is doing it, you can't really count on it as a dependable mechansim.

This feedback is non-blocking, just thinking out loud.

Reply:
> I expect VC COSE JOSE would define the same sort of mapping. IOW, just because someone asserts this, doesn't mean it's true, you need to verify it... and if only a subset of the ecosystem is doing it, you can't really count on it as a dependable mechanism.

This describes "Verifiable Presentations" as a whole... And I agree... its not very helpful.

We seem to have to do a lot of work for "holder" and "VerifiablePresentation"... when they are really just special cases of "issuer" and "VerifiableCredential".... 

I continue to assert there is a better solution to this, but I don't think we can turn this boat at this point.